packages/boot/README.md

```diff
@@ -0,0 +1,90 @@
+# @loopback/boot
+
+A collection of Booters for LoopBack Applications
+
+# Overview
+
+A Booter is a Class that can be bound to an Application and is called
+to perform a task before the Application is started. A Booter may have multiple
+phases to complete its task.
+
+An example task of a Booter may be to discover and bind all artifacts of a
+given type.
+
+A BootStrapper is needed to manage the Booters and to run them. This is packaged
+in BootComponent. Add `BootComponent` to your `Application` to use `Booters`.
+
+## Installation
+
+```shell
+$ npm i @loopback/boot
+```
+
+## Basic Use
+
+```ts
+import {Application} from '@loopback/core';
+import {ControllerBooter, RepositoryBooter, BootComponent} from '@loopback/boot';
+const app = new Application();
+app.component(BootComponent);
+app.booter(ControllerBooter); // register booter
+await app.boot({
+  projectRoot: __dirname,
+  booters: [RepositoryBooter], // Register Booters as part of call to app.boot()
+  controllers: {
+    dirs: ['controllers'],
+    extensions: ['.controller.js'],
+    nested: true
+  }
+}); // Booter gets run by the Application
+```
+
+## Available Booters
+
+### ControllerBooter
+
+#### Description
+Discovers and binds Controller Classes using `app.controller()`.
+
+#### Options
+The Options for this can be passed via `BootOptions` when calling `app.boot(options:BootOptions)`.
+
+The options for this are passed in a `controllers` object on `BootOptions`.
+
+Available Options on the `controllers` object are as follows:
+
+|Options|Type|Default|Description|
+|-|-|-|-|
+|`dirs`|`string | string[]`|`['controllers']`|Paths relative to projectRoot to look in for Controller artifacts|
+|`extensions`|`string | string[]`|`['.controller.js']`|File extensions to match for Controller artifacts|
+|`nested`|`boolean`|`true`|Look in nested directories in `dirs` for Controller artifacts|
```

Comment:
I think this list is missing an entry for `booters`?

Reply:
`booters` is a part of the `BootStrapper` config, not the `ControllerBooter` config (which is what this table represents).

Reply:
You are right!